src/www/ui/user-edit.php

```diff
@@ -55,6 +55,14 @@ protected function handle(Request $request)
     $SessionUserRec = $this->GetUserRec($user_pk);
     $SessionIsAdmin = $this->IsSessionAdmin($SessionUserRec);
 
+    if (!($SessionIsAdmin or
```

Comment:
Sorry, but I don't get what you mean. I use `$user_pk_to_modify` only in this statement and only after the line, where it will be defined. I don't think that lazy evaluation is a problem here.